Mapbox/MapboxMapsStyle/MapboxStyleURL.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-public enum StyleURL {
+public enum StyleURL: Hashable {
```

Comment:
This is a minor SDK modification which made the example easier to write.

Reply:
👍🏻